application/libraries/MY_Form_validation.php

```diff
@@ -38,11 +38,11 @@ public function has_error( $field = NULL )
          *
          * @return bool Success or Failure
          */
-//    public function run($module = '', $group = '')
-//    {
-//        is_object($module) && $this->CI =& $module;
-//        return parent::run($group);
-//    }
+        public function run($module = '', $group = '')
```

Comment:
This line makes me wonder... we have to allways use syntax run($module, group) ... but some code might have ony run($group)... this function should be rewriten to support both... or am i wrong?